### PR TITLE
Feature/rw container component

### LINKED ISSUE #23: Create Container Component for "Home" view & related view context
### Feature Categories:
- React
- Styling/CSS

**Describe the feature/Bug your gonna work on**
- Create a Container component that will conditionally render Timer, Content Selection, or Content Delivery based on the view
- Create new context for View that wraps this component, so it can be used/updated for conditional rendering 

**Additional context**
- When this is done, I can make a few quick updates on Content Delivery component to incorporate it into the Container component properly 

Add any other context or screenshots about the feature.

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
- A Home Container component was created which will hold all the different components possibly rendered on the home view (/ url) 
- View Context was also created to hold state for the string representing what the current home view is (e.g. 'timer', 'content-selection' etc)
- View Context Provider is then wrapped around Home Container component so that all its child components can access & edit the view 
- No styling was added; not sure we'll need it since this is purely a container
- A placeholder test was added; once all of the correct components are added into the Home Container, I will update the test 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
npm test 

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

resolve #23

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 import './App.css';
 import Header from '../Header/Header';
 
 function App() {
   return (
     <section className="App">
       <Header />
+      <ViewProvider>
+        <HomeContainer/>
+      </ViewProvider>
```
Comment (Abdeboskey) [inline src/Components/App/App.js:13]: I'm imagining that we will then wrap this provider in a `Route` for the `/` path, ya?

File: src/Components/ContentDelivery/ContentDelivery.js [hunk 2]
```diff
@@ -1,26 +1,44 @@
...
+        { !done &&
+          <button className={style.skipBtn} onClick={() => setView('timer')}>Skip video</button>     
+        }
+      </section>
+      { done && 
+        <div className={style.workMsgModal}>
+          <h2 className={style.modalContent}>Let's get back to work!</h2>
+        </div> 
+      }
+    </>
```
Comment (Abdeboskey) [inline src/Components/ContentDelivery/ContentDelivery.js:41]: Awesome Rachel, you're crushing the `useContext()`

File: src/Components/ContentDelivery/ContentDelivery.module.scss [hunk 3]
```diff
@@ -37,22 +42,41 @@ h2 {
...
-  .video-wrapper {
+  .videoWrapper {
     @include size(26em, 15em);
   }
 
-  .video-player {
+  .videoPlayer {
     padding: 0.5em;
   }
 }
```
Comment (Abdeboskey) [inline src/Components/ContentDelivery/ContentDelivery.module.scss:82]: This stylesheet is super fly, way to rock the includes and mixins and that blur!

File: src/Components/HomeContainer/HomeContainer.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      }
+      { view === 'content-delivery' && 
+        <ContentDelivery />
+      }
+    </VideoProvider>  
+  )
+
+}
+
+export default HomeContainer
```
Comment (Abdeboskey) [inline src/Components/HomeContainer/HomeContainer.js:28]: this is so clean! These hooks are sooo nice.

**Review by Abdeboskey — APPROVED** [review]
Awesome work Rachel!! This is gonna help things come together super quickly.

File: src/Components/HomeContainer/HomeContainer.test.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  it('should render the Timer view on load', () => {
+    const { getByRole } = render (
+      <ViewProvider>
+        <HomeContainer/>
+      </ViewProvider>
+    )
+
+    const heading = getByRole('heading', { name: 'Pom Timer View!'})
+
+    expect(heading).toBeInTheDocument();
```
Comment (sienna-kopf) [inline src/Components/HomeContainer/HomeContainer.test.js:18]: Sweet that you are testing this component!!

**Review by sienna-kopf — APPROVED** [review]

File: src/Components/ContentDelivery/ContentDelivery.module.scss [hunk 6]
```diff
@@ -37,22 +42,41 @@ h2 {
     }
 }
 
+//modal
```
Comment (jkwest-93) [inline src/Components/ContentDelivery/ContentDelivery.module.scss:45]: Love how seamless this message is looking!

File: src/Context/ViewContext.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+export const ViewProvider = ({ children }) => {
+  const [ view, setView ] = useState('timer')
+
+  return (
+    <ViewContext.Provider value={[ view, setView ]}>
+      { children}
+    </ViewContext.Provider>
+  )
+}
```
Comment (jkwest-93) [inline src/Context/ViewContext.js:13]: Great work with the ViewContext!! This is so great for how we're cycling through components.

**Review by jkwest-93 — APPROVED** [review]

**Pull request merged**